model/validate.go

```diff
@@ -90,6 +90,20 @@ func ValidateHost(hostId string, r *http.Request) (*host.Host, int, error) {
 	return h, http.StatusOK, nil
 }
 
+func ValidateContainer(containerId string, r *http.Request) (*host.Host, int, error) {
```

Comment:
The model package should just have database models and related stuff, not anything related to HTTP requests. I would just embed this logic in the parse function of the route

Reply:
Are you sure? The other functions in this file are also validations for rest routes and return http statuses. I can move it but this seems to be the current status quo.

Reply:
Yep, they really shouldn't be here but we haven't gotten around to changing them